code/modules/1713/jobs/russoukrainewar.dm

```diff
@@ -328,7 +318,7 @@
 	return TRUE
 
 /datum/job/russian/russ_sergeant
-	title = "Russian Federal Forces Sergeant "
+	title = "Russian Federal Forces Sergeant"
```

Comment:
The space is needed, or the roles wont spawn properly, they'll use rusretreat roles. Need a space at the beginning of the title (It seems I forgot to merge fixes n shit that I already made) " Russian Federal Forces Sergeant" or it wont work.

Reply:
look at japanese pacific roles for example (TBH i should remove jap pacific roles and make them an if statement in original ones

Reply:
Look at jap pacific roles for example (Which I should relocate to ww2 regular jap roles in an if statement)